packages/api/.env.example

```
@@ -28,3 +28,25 @@ ALLOW_OTP_FALLBACK=false
+# Twilio SMS Configuration (for OTP delivery)
+# Get credentials from https://console.twilio.com/
+TWILIO_ACCOUNT_SID=ACxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+TWILIO_AUTH_TOKEN=your_twilio_auth_token
+TWILIO_PHONE_NUMBER=+1XXXXXXXXXX
+
+# Optional: Twilio Verify API (recommended for production)
+# TWILIO_VERIFY_SERVICE_SID=VAxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+
+# Stripe Payment Configuration
+STRIPE_SECRET_KEY=sk_test_your_stripe_secret_key_here
+STRIPE_PUBLISHABLE_KEY=pk_test_your_stripe_publishable_key_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The STRIPE_PUBLISHABLE_KEY key should go before the STRIPE_SECRET_KEY key

(UnorderedKey)